doc/how_to/editor/vscode.md

```diff
@@ -0,0 +1,95 @@
+# Develop Apps in VS Code
+
+This guide addresses how to rapidly develop Panel applications in VS Code.
+
+We assume you have installed the VS Code Python extension and have a basic understanding of working with Python in VS Code.
+
+---
+
+## Scripts
+
+### Editing
+
+You can edit your Panel code as a ``.py`` file in VS Code, marking the objects you want to render as ``.servable()``. For example:
+
+![VS Code Script](../../_static/vscode-script.png)
+
+You launch the app with
+
+```bash
+panel serve app.py --show --autoreload
+```
+
+:::{admonition} Note
+The `--show` flag will open a browser tab with the live app and the ``--autoreload`` flag ensures that the app reloads whenever you make a change to the Python source.
+:::
+
+![App Developed in VS Code](../../_static/editor_server_app.png)
+
+### Debugging
+
+You can learn how to use the *integrated debugger* [here](https://code.visualstudio.com/docs/editor/debugging).
+
+To configure the integrated debugger for Panel, you will need to add a debugging configuration like the below.
+
+```bash
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "panel serve",
+            "type": "python",
+            "request": "launch",
+            "program": "-m",
+            "args": [
+                "panel",
+                "serve",
+                "${relativeFile}",
+                "--show"
```

Comment:
Maybe we should add `--autoreload` also. But I have no experience telling me if this is a good idea or not.